staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -427,7 +437,10 @@ func (r *crdHandler) getOrCreateServingInfoFor(crd *apiextensions.CustomResource
 	scaleScopes := map[string]handlers.RequestScope{}
 
 	for _, v := range crd.Spec.Versions {
-		safeConverter, unsafeConverter := conversion.NewCRDConverter(crd)
+		safeConverter, unsafeConverter, err := r.crdConverterFactory.NewConverter(crd)
```

Comment:
isn't one converter enough for all versions?

Reply:
This is a discussion from the previous PR (versioning PR), @liggitt wanted the converter object for each version, am I right @liggitt?

Reply:
iirc, that was the discussion at https://github.com/kubernetes/kubernetes/pull/63830#discussion_r188359702

I didn't feel strongly about having a convertor per-version. The things I thought were important were:
* construction of the convertor happened when setting up servingInfo (once per CRD change, not once per custom resource handling request)
* the convertor had an explicit list of source/target versions it would accept conversion requests for (not just a list of target versions it would stomp to)